modules/sdk/project-templates/project-templates/build.gradle

```diff
@@ -58,6 +58,8 @@ configurations {
 dependencies {
 	compileInclude group: "com.beust", name: "jcommander", version: "1.72"
 
+	compileOnly group: "org.apache.maven.archetype", name: "archetype-common", version: "2.4"
```

Comment:
Technically this isn't compileOnly because we include everything into the ProjectTemplatesJar.  I guess compileInclude could be more correct, but it doesn't really matter, since we have this https://github.com/liferay/liferay-portal/blob/master/modules/sdk/project-templates/project-templates/bnd.bnd#L10-L20

Reply:
Exactly, we can't use `compileInclude` if you only want to include certain classes and not the whole jar. Excluding `compile` (it's not really a transitive dependency), the alternative was between `provided` and `compileOnly`: but I'm slowly trying to move away from `provided`, so `compileOnly` sounded right 😊